juju-charms/charm-trilio-data-mover/src/reactive/trilio_data_mover.py

```diff
@@ -369,10 +369,13 @@ def create_service_file():
     tv_config.add_section('Install')
     tv_config.set('Unit', 'Description', 'TrilioVault DataMover')
     tv_config.set('Unit', 'After', 'openstack-nova-compute.service')
+    tv_config.set('Unit', 'StartLimitBurst', 5)
+    tv_config.set('Unit', 'StartLimitIntervalSec', 60)
     tv_config.set('Service', 'User', usr)
     tv_config.set('Service', 'Group', grp)
     tv_config.set('Service', 'Type', 'simple')
     tv_config.set('Service', 'ExecStart', exec_start)
+    tv_config.set('Service', 'RestartSec', 1)
     tv_config.set('Service', 'TimeoutStopSec', 20)
```

Comment:
@shyam-biradar , these changes in the service file will let systemd to restart the service if it fails. We don't need to restart the service from the deployment script. Will it work for RHOSP as well ?

Reply:
@abhijeetpatra : In RHOSP, we don't use systemd init mechanism, as it is container service start/stop is managed by container only.